src/storage/version/put.js

```diff
@@ -114,9 +114,19 @@ export async function putObjectWithVersion(env, daCtx, update, body, guid) {
   const pps = current.metadata?.preparsingstore || '0';
 
   // Store the body if preparsingstore is not defined, so a once-off store
-  const storeBody = !body && pps === '0';
+  let storeBody = !body && pps === '0';
   const Preparsingstore = storeBody ? Timestamp : pps;
-  const Label = storeBody ? 'Collab Parse' : update.label;
+  let Label = storeBody ? 'Collab Parse' : update.label;
+
+  if (daCtx.method === 'PUT' && current.contentLength > 83 && (!update.body || update.body.size <= 83)) {
```

Comment:
Just wondering where `83` came from?

Reply:
Size of the empty doc: https://github.com/adobe/da-collab/blob/main/src/collab.js#L143